recipes/master.rb

```diff
@@ -77,12 +76,11 @@
       action :modify
     end
 
-    include_recipe 'krb5_utils'
```

Comment:
Also, the move was intentional, since we don't really need to hit the KDC constantly with lookups. In general, anything that requires a remote service to be operational has been put under `init`, but we could move it elsewhere.